blog_app/models.py

```diff
@@ -0,0 +1,93 @@
+from django.contrib.auth.models import User
+from django.contrib.contenttypes.fields import GenericForeignKey, GenericRelation
+from django.contrib.contenttypes.models import ContentType
+from django.core.validators import RegexValidator
+from django.db import models
+
+
+class UserProfile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    user_name = models.CharField(max_length=20)
+    user_photo = models.ImageField(upload_to='photo_users', verbose_name='Photo', null=True)
+
+    user_url = models.URLField()
+    user_info = models.CharField(max_length=255, default='Information is absent')
+
+    def __str__(self):
+        return self.user_name
+
+
+class PhoneUser(models.Model):
+    phone_message = 'Phone number must be entered in the format: 0675110200'
+
+    phone_regex = RegexValidator(
+        regex=r'^(067|096|097|098|050|066|095|099|063|073|093)\d{7}$',
+        message=phone_message)
+
+    phone = models.CharField(validators=[phone_regex], max_length=10,
+                             null=True, blank=True)
+    user = models.ForeignKey(UserProfile, on_delete=models.CASCADE, related_name='phone')
+
+    def __str__(self):
+        return self.phone
+
+
+class Article(models.Model):
+    article_title = models.CharField(max_length=100, verbose_name="Title")
+    article_content = models.TextField(blank=True, verbose_name="Content")
+    article_time_create = models.DateTimeField(auto_now_add=True)
+    article_time_update = models.DateTimeField(auto_now=True)
+    article_author_id = models.ForeignKey(UserProfile,
+                                          on_delete=models.CASCADE,
+                                          related_name='author')
```

Comment:
Why did you add a related name? In this case related name 'author' - is not correct and non-readable